src/docs/resources/faq.md

```diff
@@ -866,7 +866,7 @@ In no particular order:
   geometry (for example, for cartesian boxes, a size,
   specifically a width and a height) that fulfills those constraints.
   By using this technique, Flutter can usually
-  lay out an entire scene with a single pass.
+  layout an entire scene with a single pass.
```

Comment:
"lay out" is a verb, or an action. "layout" is a noun. So the original is correct.